resources/migrations/_1427287552_MetaImage.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Message\Cog\Migration\Adapter\MySQL\Migration;
+
+class _1427287552_MetaImage extends Migration
+{
+	public function up()
+	{
+		$this->run("
+			ALTER TABLE
+				page
+			ADD
+				meta_image INTEGER DEFAULT NULL
+			AFTER
+				`type`
```

Comment:
Should be after `meta_description`, at the moment it comes before any of the other meta columns